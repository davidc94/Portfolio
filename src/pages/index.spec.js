import React from "react"
import renderer from "react-test-renderer"
import Index from "./index";

describe("Index", () => {
  it("renders as expected", () => {
    expect(renderer.create(<Index />).toJSON()).toMatchSnapshot();
  })
})