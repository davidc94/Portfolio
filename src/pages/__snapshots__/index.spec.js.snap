// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders as expected 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <h1
    className="sc-bdVaJa kUmKSc"
    size="large"
  >
    David Chellapah
  </h1>
  <h1
    className="sc-bdVaJa gFtbtq"
    size="base"
  >
    My name is David and welcome to my website 😎 I am a software developer from London (born and raised).
  </h1>
  <p
    className="sc-bwzfXH bCqCov"
    size="large"
  >
    Projects
  </p>
  <p
    className="sc-bwzfXH dGsVKy"
    size="base"
  >
    Here's a list of stuff that i've done in my life that I'm pretty proud of 👏
  </p>
  <div
    className="sc-htpNat ivMozR"
  >
    <li
      className="sc-bxivhb fxzeAI"
      size="base"
    >
      This website
    </li>
    <li
      className="sc-bxivhb fxzeAI"
      size="base"
    >
      Smart Home App (Uni dissertation)
    </li>
  </div>
  <p
    className="sc-bwzfXH dGsVKy"
  >
    Now that the boring stuff is out of the way, I also enjoy photography and shooting videos so here's a small collection of of videos and pictures from travelling ✈️.
  </p>
  <a
    href="https://github.com/davidc94"
    target="_blank"
  >
    <div
      className="sc-htpNat cijKQG"
      style={
        Object {
          "margin": "8px",
        }
      }
    >
      <img
        className="sc-ifAKCX eXPugd"
        src=""
      />
    </div>
  </a>
  <a
    href="https://www.linkedin.com/in/davidchellapah/"
    target="_blank"
  >
    <div
      className="sc-htpNat cijKQG"
    >
      <img
        className="sc-ifAKCX eXPugd"
        src=""
      />
    </div>
  </a>
</div>
`;
